README.md

```diff
@@ -307,6 +307,8 @@ deployment:
       - goreleaser
 ```
 
+(Note that if you test multiple versions or multiple OSes you probably want to make sure GoReleaser is just run once)
```

Comment:
Not sure if it's a good fit since it's not an actual quote.
What do you think about making it italic?